galoy.yaml

```diff
@@ -199,3 +199,6 @@ captcha:
   mandatory: false
 skipFeeProbe:
   - 038f8f113c580048d847d6949371726653e02b928196bad310e3eda39ff61723f6
```

Comment:
why keep this and maintain some `Legacy` code? we don't need backward compatibility, we can just update the associated yaml? just make sure to have the ! in the commit message to signal a breaking change.